components/operations-controller/internal/webhook/client.go

```diff
@@ -206,6 +206,9 @@ func parseResponseObject(resp *http.Response) (*web_hook.ResponseObject, error)
 		}
 
 		for k, v := range tmpBody {
+			if v == nil {
```

Comment:
extend the unit tests in client.go and the e2e tests in external service async delete (the e2e test is already there, it just returns not the real response body) to cover this case